src/haversine.jl

```diff
@@ -33,6 +34,6 @@ function (dist::Haversine)(x::VecOrLengthTwoTuple, y::VecOrLengthTwoTuple)
     2 * dist.radius * asin( min(√a, one(a)) ) # take care of floating point errors
 end
 
-haversine(x::VecOrLengthTwoTuple, y::VecOrLengthTwoTuple, radius::Real) = Haversine(radius)(x, y)
+haversine(x::VecOrLengthTwoTuple, y::VecOrLengthTwoTuple, radius::Real = 6378137.0) = Haversine(radius)(x, y)
```

Comment:
Convention in this package seems to be to use no spaces:
```suggestion
haversine(x::VecOrLengthTwoTuple, y::VecOrLengthTwoTuple, radius::Real=6378137.0) =
    Haversine(radius)(x, y)
```